README.rst

```diff
@@ -294,6 +294,16 @@ by default the data is added to the first found organization, eg::
 
 **Note**: you can follow the `tutorial to migrate database from django-netjsongraph <https://github.com/openwisp/django-netjsongraph/blob/master/README.rst>`_.
 
+create_device_nodes
+^^^^^^^^^^^^^^^^^^^
```

Comment:
always look at how the documentation you write looks: https://github.com/openwisp/openwisp-network-topology/tree/issue-92-DeviceNode-management-command#upgrade_from_django_netjsongraph

See the difference?

Needs to be:

```
``create_device_nodes``
^^^^^^^^^^^^^^^^^^^^^^^
```

BTW, also look at this: https://github.com/openwisp/openwisp-network-topology#openwisp-network-topology-api-auth-required, I didn't spot it before.. please stay consistent with the rest of the codebase.

Reply:
@nemesisdesign you were the one who discouraged double ticks from headings in one PR I send. see https://github.com/openwisp/openwisp-network-topology/pull/84#discussion_r471816966

Reply:
Read again: 

> do not use if for non-code text please

`create_device_nodes` is code.

Reply:
Ok thanks